support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -197,7 +197,8 @@ object SendAcquisitionEvent {
         Some(Set(
           if (stateAndInfo.requestInfo.accountExists) Some("REUSED_EXISTING_PAYMENT_METHOD") else None,
           if (isSixForSix(stateAndInfo)) Some("guardian-weekly-six-for-six") else None,
-          stateAndInfo.state.giftRecipient.map(_ => "gift-subscription")
+          stateAndInfo.state.giftRecipient.map(_ => "gift-subscription"),
+          stateAndInfo.state.paymentMethod.right.map(_ => "corporate-subscription").toOption
```

Comment:
if we move the lib into support-frontend as discussed, we can use a constant so I don't have to check for typos in the labels